src/main/java/com/storozh/SpringExample/controller/StudentController.java

```diff
@@ -0,0 +1,49 @@
+package com.storozh.SpringExample.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import com.storozh.SpringExample.DTO.StudentDTO;
+import com.storozh.SpringExample.model.Student;
+import java.util.List;
+import com.storozh.SpringExample.service.StudentService;
+
+@RestController
+@RequestMapping(value="/api/student")
+public class StudentController {
+
+    @Autowired
+    StudentService studentService;
+
+    @PostMapping
+    public String create(@RequestBody StudentDTO studentDTO) {
+        studentService.createStudent(studentDTO);
+        return "Student is created";
+    }
+
+    @GetMapping
+    public List<Student> getAllStudents() {
+        return studentService.getAllStudents();
+    }
+
+    @PutMapping("/update/{id}")
+    public ResponseEntity<Student> updateStudent(@PathVariable Long id, @RequestBody StudentDTO studentDTO) {
+        Student student = studentService.updateStudent(id, studentDTO);
+        if (student == null) {
+            return ResponseEntity.notFound().build();
```

Comment:
in java 8 you can use Optional to check null value and throw exception. (use RuntimeException)